### PR TITLE
added gertzakis's hello script

### PR DESCRIPTION
**Pull request by gertzakis**

### CODE REVIEW

File: scripts/99_from_makis.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+print("Hello There!")
+print("Hello again")
+print("HELLOOOOS")
+print("another commit to check merge")
```
Comment (cmsirbu) [inline scripts/99_from_makis.py:4]: This is not a hello line, would you please remove it? :)

**Review by cmsirbu — CHANGES_REQUESTED** [review]

File: scripts/99_from_makis.py (hunk 1 above)
Comment (gertzakis) [reply scripts/99_from_makis.py]: Is this OK now? :)

Comment (cmsirbu) [reply scripts/99_from_makis.py]: 👍🏻 thanks!

**Review by cmsirbu — APPROVED** [review]
Looks good, merging!

**Pull request merged**